tests/starwars/test_dsl.py

```diff
@@ -387,27 +384,25 @@ def test_operation_name(ds):
 
 def test_multiple_operations(ds):
     query = dsl_gql(
-        GetHeroName=DSLQuery(ds.Query.hero.select(ds.Character.name)),
-        CreateReviewMutation=DSLMutation(
+        DSLQuery(ds.Query.hero.select(ds.Character.name)),
+        DSLMutation(
             ds.Mutation.createReview.args(
                 episode=6, review={"stars": 5, "commentary": "This is a great movie!"}
             ).select(ds.Review.stars, ds.Review.commentary)
         ),
     )
 
+    result = print_ast(query)
     assert (
-        print_ast(query)
-        == """query GetHeroName {
+        result
+        == """{
```

Comment:
Similar issue here, `test_multiple_operations` intended to test presence of `GetHeroName`.

Reply:
:+1: